docs/storybook.md

```diff
@@ -0,0 +1,45 @@
+# Storybook for Open Sauced
+
+Storybook is being leveraged to mock out visual React components. The latest version of the design system can be found at this [URL](https://sauced-components.netlify.app/).
+
+To run storybook, use this command:
+
+```sh
+npm run storybook
+```
+
+## UI Categories
+
+Storybook is broken into several categories:
+
+<ul>
+<li><b>Button:</b> These are the button elements that appear in the project in various forms. They primarily are the Button component in the project but can also be icons.</li>
+<li><b>Cards:</b> These are the main container elements in the project. Each item represents a live component in their current form in the project.</li>
+<li><b>Primitives: These are the basic styling of base HTML components.</b></li>
+<li><b>Nav:</b> This is the main navigation bar for the project. There are two states, when there is no user logged in and when a user is logged in.</li>
+<li><b>Footer:</b> This represents the various footers for the project.</li>
+<li><b>Homepage:</b> This is the main component for the project homepage and shows the home page in its current form.</li>
+<li><b>Miscellaneous:</b> These are components that currently don't fit neatly into the above categories.</li>
+</ul>
+
+## Making changes to Storybook
```

Comment:
Are you still working on this? I think this is close to good, just saw this section was empty..

Reply:
Oh sorry. It was meant to be a header for the section. I made some changes to the docs to make it a bit more clear and made the necessary formatting changes to the other headings under it.